src/server/scripts/Commands/cs_guild.cpp

```diff
@@ -299,6 +302,60 @@ class guild_commandscript : public CommandScript
         handler->PSendSysMessage(LANG_GUILD_INFO_EXTRA_INFO, guild->GetInfo().c_str()); // Extra Information
         return true;
     }
+
+    static bool HandleGuildListCommand(ChatHandler* handler, char const* /*args*/)
+    {
+        auto const& guildStore = sGuildMgr->GetGuildStore();   // map<uint32, unique_trackable_ptr<Guild>>
```

Comment:
We don't use `auto` except for long nested template types like container iterators